mobile/library/common/mobile_process_wide.cc

```diff
@@ -0,0 +1,13 @@
+#include "library/common/mobile_process_wide.h"
+
+namespace Envoy {
+
+MobileProcessWide::MobileProcessWide(const OptionsImplBase& options) {
+  logging_context_ = std::make_unique<Logger::Context>(options.logLevel(), options.logFormat(),
```

Comment:
If we have multiple Envoy Mobile engines, does that mean setting the log level will apply globally to all Envoy Mobile engines?

Reply:
Yes, the way `Context` works now (with a stack of Contexts), this was the only way to initialize logging and make it safe from data races in EM.

We probably need to rework how we do initialization of globals in general and `Logger::Context` in specific, but that's outside the scope of this PR.